CODEOWNERS

```diff
@@ -1,5 +1,168 @@
 # CODEOWNERS for autoreview assigning in github
 
 # Get all docs reviewed
-doc/	@dbkinder
-*.rst	@dbkinder
+
+*                                        kumar.gala@linaro.org anas.nashif@intel.com
+*.rst                                    @dbkinder
+.known-issues/*                          inaky.perez-gonzalez@intel.com anas.nashif@intel.com
+MAINTAINERS                              inaky.perez-gonzalez@intel.com anas.nashif@intel.com
+arch/arc/*                               cjordan@synopsys.com Ruud.Derwig@synopsys.com
+arch/arc/core/*                          andrew.p.boie@intel.com
+arch/arm/*                               maureen.helm@nxp.com kumar.gala@linaro.org
+arch/arm/core/*                          andrew.p.boie@intel.com
```

Comment:
This is obviously not the final change to this file, we need to start somewhere and optimise and modify as we go. and yes, you are right on this. In some cases this make sense, but in others, it does not ( I am not going to add myself to every entry because on the top of the file I own *)